docs/troubleshooting.md

```diff
@@ -3,34 +3,35 @@
 ## Table of Contents
 
 - [How to debug the Scanner VMs](#how-to-debug-the-scanner-vms)
-  - [AWS](#debug-scanner-VM-on-AWS)
+  - [Docker and Kubernetes](#docker-and-kubernetes-provider)
+  - [Cloud providers](#cloud-providers)
 
 ## How to debug the Scanner VMs
 
-How to debug the Scanner VMs can differ per provider these are documented
-below.
+### Docker and Kubernetes provider
 
-### Debug Scanner VM on AWS
+In case of the Docker provider scanners are created as containers, and as pods on Kubernetes so they can be accessed
```

Comment:
```suggestion
For Docker provider, scanners are created as containers, while as pods in case of Kubernetes. In both cases, you can access them directly and check the logs.```